AUTHORS

```diff
@@ -0,0 +1,11 @@
+# This is the list of vscode-interactive-graphviz's significant contributors.
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
+# To see the full list of contributors, see the revision history in
+# source control.
+#
+# Format: <name> [(url1) (urlN)]
+#
+tintinweb (http://github.com/tintinweb)
+bigbug (http://github.com/bigbug)
```

Comment:
@bigbug feel free to add more ext. links here if you want 🙏

Reply:
Seems sufficient for me :)